.github/workflows/ci.yml

```diff
@@ -58,33 +58,39 @@ jobs:
           npx playwright install --with-deps
       - name: Run tests
         run: npm run test:all
+      - name: Upload coverage report
+        id: coverage-report
+        uses: actions/upload-artifact@main
```

Comment:
Not use our version since it's not up to date and is missing features. Since this is a public repo this doesn't incur cost.